cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -189,3 +217,49 @@ def test_api_v1_check_duplicated_polygon_points(self):
                 polygon_points = polygon.attrib["points"].replace(",", ";")
                 polygon_points = [float(p) for p in polygon_points.split(";")]
                 self.assertEqual(polygon_points, annotation_points)
+
+    def test_api_v1_check_widerface_with_all_attributes(self):
+        test_name = self._testMethodName
+        dump_format_name = "WiderFace 1.0"
+        upload_format_name = "WiderFace 1.0"
+
+        for include_images in (False, True):
+            with self.subTest():
+                # create task with annotations
+                images = self._generate_task_images(3)
+                task = self._create_task(tasks["widerface with all attributes"], images)
```

Comment:
In the future here it is better to use "backup" functionality to create a task. Also I don't like that all our tests cases in one file. Probably they should be spitted. What do you think?

Reply:
@nmanovic , yes, we probably need to have one file with the main functions for the tests and separate files for the tests.